service/agentconf.go

```diff
@@ -94,30 +94,19 @@ func NewServiceManager(
 // WriteServiceFiles writes service files to the standard
 // /etc/systemd/system path.
 func (s *systemdServiceManager) WriteServiceFiles() error {
-	machineAgent, unitAgents, _, err := s.FindAgents(paths.NixDataDir)
+	machineAgent, _, _, err := s.FindAgents(paths.NixDataDir)
```

Comment:
Maybe a note on `FindAgents` about retention of the unit discovery for upgrade steps?

Reply:
ok.